src/modules/filesystem/NativeFile.cpp

```diff
@@ -264,32 +193,7 @@ bool NativeFile::setBuffer(BufferMode bufmode, int64 size)
 	if (bufmode == BUFFER_NONE)
 		size = 0;
 
-	// If the file isn't open, we'll make sure the buffer values are set in
-	// NativeFile::open.
-	if (!isOpen())
-	{
-		bufferMode = bufmode;
-		bufferSize = size;
-		return true;
-	}
-
-	int vbufmode;
-	switch (bufmode)
-	{
-	case File::BUFFER_NONE:
-	default:
-		vbufmode = _IONBF;
-		break;
-	case File::BUFFER_LINE:
-		vbufmode = _IOLBF;
-		break;
-	case File::BUFFER_FULL:
-		vbufmode = _IOFBF;
-		break;
-	}
-
-	if (setvbuf(file, nullptr, vbufmode, (size_t) size) != 0)
-		return false;
+	// FIXME: SDL doesn't have option to set buffering.
```

Comment:
Maybe we should make an issue in their repo about this? I feel like it's pretty useful / important, but at the very least we'd get their opinion.

Reply:
That being said, for now we could probably implement `BUFFER_LINE` ourselves (like what's done in physfs/File.cpp) and leave the rest as a todo.

Reply:
Alright I opened ticket in SDL for this. libsdl-org/SDL#11832

Reply:
Considering that SDL is just low-level wrapper to OS-specific API (such as Windows file `HANDLE` or POSIX file descriptor) and there's no explicit way to set buffering mode on those handle/file descriptors, then the best but unfortunate way to solve this is to implement our own buffering.